src/routers/sessions.py

```
@@ -460,12 +462,24 @@
 async def search_session(
     workspace_id: str = Path(..., description="ID of the workspace"),
     session_id: str = Path(..., description="ID of the session"),
-    query: str = Body(..., description="Search query"),
+    search: schemas.MessageSearchOptions = Body(
+        ..., description="Message search parameters "
+    ),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)